### PR TITLE
expose signer through getValidator

### PR DESCRIPTION
**Pull request by diminator**
`getValidator(validator)` now also returns `signer:address`. required for ethstats valset management

### CODE REVIEW

File: packages/protocol/test/governance/validators.ts [hunk 1]
```diff
@@ -1091,6 +1098,7 @@ contract('Validators', (accounts: string[]) => {
           })
 
           it('should set the validator ecdsa public key', async () => {
+            await registry.setAddressFor(CeloContractName.Accounts, accountsInstance.address)
```
Comment (timmoreton) [inline packages/protocol/test/governance/validators.ts:1101]: why do you need this?

**Review by timmoreton — APPROVED** [review]

**Comment by timmoreton** [comment]
Have you tested CLI and other users of `getValidator` works with these changes ? should update `election/current.ts` etc.

File: packages/protocol/test/governance/validators.ts (hunk 1 above)
Comment (diminator) [reply packages/protocol/test/governance/validators.ts:1101]: `getValidator` calls `getAccount().getValidatorSigner` account.address needs to be contract, not wallet

Comment (diminator) [reply packages/protocol/test/governance/validators.ts:1101]: https://github.com/celo-org/celo-monorepo/pull/1997/files/f308cff7a9aa1a716337f18747382358a7984b2f#diff-cee92562946fb3c6fc71ec4d162da770R1087

**Comment by diminator** [comment]
> Have you tested CLI and other users of `getValidator` works with these changes ? should update `election/current.ts` etc.

haven't exposed this to the entire API. only for contract_comm due to time reasons

**Pull request merged**